Gemfile

```diff
@@ -62,7 +62,7 @@ gem "paper_trail", "~> 10.2.0"
 # Use the waste exemptions engine for the user journey
 gem "waste_exemptions_engine",
     git: "https://github.com/DEFRA/waste-exemptions-engine",
-    branch: "master"
+    branch: "feature/transient-reg-subclasses"
```

Comment:
I don't think this should be part of the changes right? :)

Reply:
Nope, I'll have to rip it out once the engine update is merged!